packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -988,11 +997,19 @@ export function renderDidError() {
   }
 }
 
-function inferTimeFromExpirationTime(expirationTime: ExpirationTime): number {
+function inferTimeFromExpirationTime(
+  expirationTime: ExpirationTime,
+  suspenseConfig: null | SuspenseConfig,
+): number {
   // We don't know exactly when the update was scheduled, but we can infer an
   // approximate start time from the expiration time.
   const earliestExpirationTimeMs = expirationTimeToMs(expirationTime);
-  return earliestExpirationTimeMs - LOW_PRIORITY_EXPIRATION;
+  return (
+    earliestExpirationTimeMs -
```

Comment:
Let's store the start time on the batch config so we don't have to infer it? Can do in a follow up.

Reply:
However the start time computation is pretty broken right now and it might be nice to get it soon so we can rule it out as a source of possible wonkiness when we get bug reports.

Reply:
I don't want to have to create a new object for the batch too.

Reply:
Can it not be the same object as `suspenseConfig`?

Reply:
`suspenseConfig` is a user space object so mutating it gets fragile. I also expect it to mostly be a hoisted shared object. E.g. the default one right now but also the defaults in hooks would just use a hoisted config.

We could have another field for that but that requires me to track another thing for the render path. It's probably the right strategy eventually but I think we need to think about it some more.